src/proto/FunctionRpc.proto

```diff
@@ -698,3 +701,30 @@ message ModelBindingData
 message CollectionModelBindingData {
   repeated ModelBindingData model_binding_data = 1;
 }
+
+// Retry policy which the worker sends the host when the worker indexes
+// a function.
+message RpcRetryOptions
+{
+  // The retry strategy to use. Valid values are fixed delay or exponential backoff.
+  enum RetryStrategy
+  {
+      exponential_backoff = 0;
+      fixed_delay = 1;
+  }
+
+  // The maximum number of retries allowed per function execution.
+  //  -1 means to retry indefinitely.
+  int32 max_retry_count = 2;
+
+  // The delay that's used between retries when you're using a fixed delay strategy.
+  google.protobuf.Duration delay_interval = 3;
```

Comment:
More info on duration type can be found [here](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/duration.proto). We should be able to manually build this message if there's no helper method already available. Dotnet has a helper method that converts from `TimeSpan` and python also has a helper method.

Reply:
@ejizba @Francisco-Gamino Since your language workers also have worker-indexing models, can you confirm this type is compatible? I built the message without the helper and it worked for me in dotnet, but wanted to make sure you guys saw this.

Reply:
Looks like we use `Duration` elsewhere in here already... but good idea to confirm with everyone. It's possible other workers don't use worker terminate:

```
message WorkerTerminate {
  google.protobuf.Duration grace_period = 1;
}
```

Reply:
@brettsam I also considered that for justification but @gavin-aguiar said he doesn't believe Python uses it and the only mention of it in their repo is the protobuf file ...

Reply:
Confirmed that duration can build in node just fine and I believe powershell would work as well.